doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,9 +63,385 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
 Depends xxx.
 
 <a name="6.3"></a>
-## 6.3 Descriptor file definition
+## 6.3 Descriptor file definition specification
```

Comment:
In generally, for each part of the descriptor, should we specify the number of instances, i.e. some might be 1 or more, some might be 0 or more, and some could be 1 instance (singleton).

Reply:
Good suggestion, will update